index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Sajjda Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <section>
+    <header id="main-header">
+      <h2 id="logo">Sajjad</h2>
+      <img src="images/menu-icon.png" id="menu-icon" alt="menu icon">
+    </header>
+    <article>
+      <div id="primary-text">I’m Sajjad Glad to see you!</div>
+      <div id="secondary-text">I’m a software developer! I can help you build a product , feature or website Look
+        through some of my work and experience! If you like what you see and have a project you need coded, don’t
+        hestiate to contact me.</div>
+      <div id="connect">LET'S CONNECT</div>
+      <div id="social">
+        <!-- <i class="fa fa-twitter"></i>
+        <i class="fa fa-linkedin"></i>
+        <i class="fa fa-facebook"></i>
+        <i class="fa fa-youtube"></i>
+        <i class="fa fa-vector"></i> -->
```

Comment:
please delete this comment code to make your code clean and your file professional